### DISCUSSION TITLE
pause argocd, set it not to sync any apps for a while

### DISCUSSION THREAD

**Discussion by vvatlin** [argoproj/argo-cd#8396] [General]
Hello. 
I want to check several ideas on my test cluster, but argocd rollback my changes, is it possible to pause the whole argocd (not the one app) for a while ?

**Comment by chetan-rns** [comment]
Hi @vvatlin, please check sync windows https://argo-cd.readthedocs.io/en/stable/user-guide/sync_windows/. It can be configured at the `Project` level targeting multiple apps.

**Comment by vvatlin** [comment]
it works. thank you
